### ISSUE TITLE
bug: page footer expansion obstructs text and UI/UX buttons hidden behind it

### ISSUE THREAD

**Issue by silentoplayz** [open-webui/open-webui#2231]
# Bug Report

## Description

**Bug Summary:**

It appears the entire footer is expanded with the messagebox itself, hence responses being cut off in chats along with buttons hidden behind the expanded footer.

**Steps to Reproduce:**
2. Choose a local or external large language model in the model dropdown to begin chatting with.
3. Send enough queries to where a scrollbar is created & required for the chat itself.
4. Paste in some chunky text (enough lines) in the message box.
5. The entire footer area appears to expand with the messagebox, resulting in the text and UI/UX buttons being cut off from being displayed on the screen correctly.

**Expected Behavior:**

- The footer should not expand in size when the messagebox expands in size.

**Actual Behavior:**

- When displaying a message box, the text content is truncated and buttons are not made fully visible.

## Environment

- **Open WebUI Version:** v0.1.124 [(latest)](https://github.com/open-webui/open-webui/releases/tag/v0.1.124)
- **Ollama:** [v0.1.37](https://github.com/ollama/ollama/releases/tag/v0.1.37)
- **Operating System:** Windows 11 Pro Insider Preview (Edition) - Version: 22H2 - Installed on: February 4, 2024 - OS build: 23620.1000 - Experience: Windows Feature Experience Pack 1000.23620.1000.0
- **Browser:** Firefox v125.0.3 (64-bit)

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.

## Logs and Screenshots

![image](https://github.com/open-webui/open-webui/assets/50341825/57a6978a-c0e5-4542-8cfb-0b7b3b250e6e)
![image2](https://github.com/open-webui/open-webui/assets/50341825/3f647e51-79d7-4c5b-8a58-1dff6d00f6eb)
![image3](https://github.com/open-webui/open-webui/assets/50341825/08968aaf-5f3e-446f-bea9-396e80431072)

**Browser Console Logs:**
- N/A

**Docker Container Logs:**
- N/A

## Installation Method

Docker Desktop via docker-compose.yml file for Open WebUI, Ollama is running natively on Windows with the Windows (Preview) version.

## Additional Information

- Attaching an image causes Llama 3's response to be pushed higher up on the screen, but no additional information is revealed.

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
Closing in favour of https://github.com/open-webui/open-webui/issues/2504

**Comment by gamepad-coder** [comment]
I came to report the same thing.

However since there's a workaround on desktop, but no workaround on mobile for Firefox (always totally broken right now), I created a dedicated issue to CSS patch mobile here:
- [#2587 bug: CSS patch needed, Android Firefox totally unable to click response buttons](https://github.com/open-webui/open-webui/issues/2587)

---

> [!NOTE]
>
> The patch that this `#2231` was closed in favor of
>
> - [#2504 ui: Scrollbar for the chat conversations goes behind the Message Box](https://github.com/open-webui/open-webui/issues/2504)
>
> may or may not also resolve my `#2587` above, depending on how it's implemented.
>
>So I'd recommend keeping `#2587` open until we confirm that basic mobile functionality is unblocked.

**Comment by gamepad-coder** [comment]
✅  [Totally patched](https://github.com/open-webui/open-webui/issues/2504#issuecomment-2143654390) 🎉
